CHANGELOG.md

```diff
@@ -136,6 +136,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Deprecated LightningCLI's registries in favor of importing the respective package ([#13221](https://github.com/PyTorchLightning/pytorch-lightning/pull/13221))
 
 
+
+- Deprecated `pytorch_lightning.profiler` in favor of `pytorch_lightning.profilers` ([#12308](https://github.com/PyTorchLightning/pytorch-lightning/pull/12308))
```

Comment:
The commits are not correct: https://github.com/Lightning-AI/lightning/pull/12308/commits/29d6cdaad861f533e9604bbd75ec73ea3b856177 should only be modifying the filenames (breaking compatibility) and the next one just adding back compatibility for the old name

Reply:
What am I supposed to do